tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -483,6 +484,12 @@ void ordering_test_cases() {
 
         spaceship_test<std::strong_ordering>(c_mem[0], c_mem[0], c_mem[1]);
     }
+    { // slice
+        static_assert((std::slice(2, 3, 4) == std::slice(2, 3, 4)) == true);
+        static_assert((std::slice(2, 3, 4) == std::slice(3, 3, 4)) == false);
+        static_assert((std::slice(2, 3, 4) == std::slice(2, 4, 4)) == false);
+        static_assert((std::slice(2, 3, 4) == std::slice(2, 3, 3)) == false);
+    }

```

Comment:
A couple of stylistic suggestions:

* Naming the slices would be somewhat more consistent with the other tests here, and would avoid some repetition, e.g. `std::slice s1(2, 3, 4);`.
* We can rely on C++20's rewriting of `!=`. That is, instead of testing `(A == B) == true` and `(A == C) == false`, we can simply test `A == B` and `A != C`, like `space_info` does immediately below.